tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -77,6 +80,33 @@ <h3>No profile data was found.</h3>
   <tf-dashboard-layout>
   <div class="sidebar">
     <div class="allcontrols">
+      <div class="sidebar-section">
+        <paper-button raised onclick="captureProfileDialog.open()">Capture Profile</paper-button>
```

Comment:
I might be misremembering but please refrain from using JS in an attribute like this. It is more prone for mistake that at the worst can cause XSS (and harder to statically analyze -- i don't want HTML AST parser just for this!).

First, please use `on-tap` (or `on-click`; [Polymer specific] tap is preferred because it works both for touch and mouse devices!) with a callback as string (https://polymer-library.polymer-project.org/1.0/docs/devguide/events)